shared/common-adapters/standard-screen.native.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import {Component} from 'react'
+import type {Props} from './standard-screen'
+
+class StandardScreen extends Component<void, Props, void> {
```

Comment:
Ahh, yes. I haven't looked to see if there is an analogous component on the native side. Maybe something like these screens: zpl.io/2fHhmY, zpl.io/Z236D2Y, zpl.io/181w45